src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -536,6 +545,13 @@ class DevTask extends AbstractFeatureTask {
             boolean installFeatures = false;
             boolean optimizeGenerateFeatures = false;
 
+            if (isMultiModuleProject()) {
+                info("We detected a change in build.gradle, but we cannot identify whether it's a runtime or dependency change");
+                info("r - to restart server, type 'r' and press Enter.");
+                info("Press Enter to recompile the project and run tests manually");
+                return false;
```

Comment:
"A change was detected in a build file. The libertyDev task could not determine if a server restart is required. To restart the server, type 'r' and press Enter."

I don't think we want to tell them about running tests here. If we do, it needs special logic to handle the fact that the key could have been changed from Enter to 't' plus Enter.

Reply:
Message corrected